### PR TITLE
Improve unit tests

### PR DESCRIPTION
**Pull request by fvaleri**
- Removed unnecessary @QuarkusTest annotation from all unit tests (this is a lot faster)
- Renamed unit tests names to better reflect the actual class under test
- Renamed the original health test as integration test b/c it starts Quarkus runtime

### CODE REVIEW

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Review by Frawless — APPROVED** [review]

File: src/test/java/io/strimzi/HealthCheckIT.java [hunk 1]
```diff
@@ -7,7 +11,7 @@
 import static org.hamcrest.CoreMatchers.is;
 
 @QuarkusTest
-public class HealthCheckResourceTest {
+public class HealthCheckIT {
```
Comment (scholzj) [inline src/test/java/io/strimzi/HealthCheckIT.java:14]: I thought we wanted to keep this as regular unit test? I'm not entirely sure I understand why we need this and HealthCheckTest.

**Review by scholzj — COMMENTED** [review]
Could you please rebase it and update after we merged the system tests? It changed a bit the structure of packages -> all unit tests (and integration tests?) are now in the `io.strimzi.test` package.

File: src/test/java/io/strimzi/HealthCheckIT.java (hunk 1 above)
Comment (fvaleri) [reply src/test/java/io/strimzi/HealthCheckIT.java:14]: I created both. Although simple, I still think the original one should be classified as integration test as it requires to start the Quarkus runtime, which is an overkill for a unit test.

**Comment by fvaleri** [comment]
Sure, I will rebase.

**Comment by fvaleri** [comment]
Rebased and tested both unit and system tests.

**Pull request merged**